tutor/src/screens/assignment-review/ux.js

```diff
@@ -254,4 +254,8 @@ export default class AssignmentReviewUX {
     return Boolean(['reading', 'homework'].includes(this.planScores.type));
   }
 
+  @computed get canDisplaySubmissionAndScoresTabs() {
+    return Boolean(['reading', 'homework'].includes(this.planScores.type));
+  }
+
```

Comment:
@jivey isn't this returning the same thing as the function above?

Reply:
@camliu89 I thought about DRYing them into a single method but couldn't decide on a good name to refer to these two types. "Workable"? Let me know if you have any suggestions!

Reply:
mmm. I agree with you. I don't like long names. mmm. "hasReadingOrHomeworkType"? It is what is checking for :thinking:

Reply:
That's way better and simpler! 😄